### PR TITLE
Get info from Backend in PollActivity

### LINKED ISSUE #30: We need to get the id of the session with get /session

### LINKED ISSUE #37: Get Poll from backend

### PR DESCRIPTION
**Pull request by Stellucidam**
We want to get the poll information (eg questions, answers...) when we start the poll activity.

### CODE REVIEW

**Comment by Stellucidam** [comment]
Not done very elegantly for now but it works.

File: app/src/main/java/ch/heigvd/pro/b04/android/Poll/PollActivity.java [hunk 1]
```diff
@@ -7,45 +7,71 @@
...
+import ch.heigvd.pro.b04.android.network.RockinAPI;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
 
 public class PollActivity extends AppCompatActivity {
     private PollViewModel state;
+    private Token token;
+
+    private Callback<Poll> callbackPoll = new Callback<Poll>() {
```
Comment (glsubri) [inline app/src/main/java/ch/heigvd/pro/b04/android/Poll/PollActivity.java]: Should not be in activity

File: app/src/main/java/ch/heigvd/pro/b04/android/datamodel/Session.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
+package ch.heigvd.pro.b04.android.datamodel;
+
+import com.google.gson.annotations.SerializedName;
+
+public class Session {
```
Comment (glsubri) [inline app/src/main/java/ch/heigvd/pro/b04/android/datamodel/Session.java:5]: We do not need a Session object

File: app/src/main/java/ch/heigvd/pro/b04/android/datamodel/Poll.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    private String idPoll;
+
+    @SerializedName("idSession")
+    private String idSession;
+
+    @SerializedName("code")
+    private String code;
+
+    @SerializedName("status")
+    private String status;
```
Comment (glsubri) [inline app/src/main/java/ch/heigvd/pro/b04/android/datamodel/Poll.java]: We do not need the status

File: app/src/main/java/ch/heigvd/pro/b04/android/network/RockinAPI.java [hunk 4]
```diff
@@ -1,13 +1,24 @@
...
 
 public interface RockinAPI {
 
     @POST("/connect")
     Call<Token> postConnect(@Body SessionCode code);
+
+    @GET("/session")
+    Call<Session> getSession(@Query("token") String userToken);
+
+    @GET("/mod/{mod}/poll/{poll}")
```
Comment (glsubri) [inline app/src/main/java/ch/heigvd/pro/b04/android/network/RockinAPI.java]: Can we keep the same format as in https://github.com/heig-PRO-b04/java-backend/wiki/APIv1.Objects ?

File: app/src/main/java/ch/heigvd/pro/b04/android/datamodel/Session.java (hunk 2 above)
Comment (Stellucidam) [reply app/src/main/java/ch/heigvd/pro/b04/android/datamodel/Session.java:5]: We need i tfor now, in order to get the Poll.

**Review by glsubri — DISMISSED** [review]

**Review by glsubri — APPROVED** [review]

**Pull request merged**